awx/ui_next/src/App.jsx

```diff
@@ -113,19 +128,17 @@ function App() {
           <Route exact strict path="/*/">
             <Redirect to={`${pathname.slice(0, -1)}${search}${hash}`} />
           </Route>
-          <Route path="/login">
-            <Login isAuthenticated={isAuthenticated} />
-          </Route>
-          <Route exact path="/">
-            <Redirect to="/home" />
-          </Route>
-          <ProtectedRoute>
-            <ConfigProvider>
-              <AppContainer navRouteConfig={getRouteConfig()}>
-                <AuthorizedRoutes routeConfig={getRouteConfig()} />
-              </AppContainer>
-            </ConfigProvider>
-          </ProtectedRoute>
+          <ConfigProvider>
+            <Route path="/login">
```

Comment:
This change was required to allow Login screen to have access to the values provided by the ConfigProvider, in particular the request to fetch data.